Get-help.md

```diff
@@ -0,0 +1,141 @@
+Here we document Oppia's main communication channels and how to ask good questions.
+
+## Communication channels
+
+If you need help, there are a few communication channels you can use. Developers usually respond within 24 hours so long as you use a channel they actually check.
+
+### Email
+
+We have several mailing lists in the form of Google Groups that you can join:
+
+* [oppia-announce](https://groups.google.com/forum/#!forum/oppia-announce) is for announcements of new releases or blog posts. It's not for asking questions though.
+* [oppia-dev](https://groups.google.com/forum/#!forum/oppia-dev) is the main mailing list for communication between developers and for technical questions. You can post to it even if you're not a member of the group. This is where you can ask questions, solicit feedback, or make developer-specific announcements (e.g. a temporary GitHub outage).
+
+You can also email your mentor with any questions. If you don't have a mentor, complete the steps on the [[wiki page for contributing code to Oppia|Contributing-code-to-Oppia]].
```

Comment:
Perhaps end with "... and you'll be assigned one"?

Reply:
done